src/main/java/com/agilethought/internship/sso/controller/GlobalExceptionHandler.java

```diff
@@ -1,44 +0,0 @@
-package com.agilethought.internship.sso.controller;
```

Comment:
Don't remove this class, it belongs to previuos behaviors, you need to adapt your code to this handler